src/d/actor/d_a_tag_attention.cpp

```diff
@@ -37,42 +40,120 @@ static dCcD_SrcSph sph_check_src = {
     },
 };
 
+bool daTagAttention::Act_c::chk_inside(cXyz* pos) const {
+    if (!m_b0x290){
+        return false;
+    }
+
+    cXyz plyrToObjVec = dComIfGp_getPlayer(0)->current.pos - current.pos;
+    if (subtype == 0){
+        // spherical collision check
+        f32 distance = plyrToObjVec.getSquareMag();
+        distance = std::sqrtf(distance);
+        
+        if (distance > scale.x * 100.0f) {
+            return false;
+        }
+    }else{
+        // box collision check
+        s16 yRotAngle = current.angle.y;
+        if (yRotAngle != 0){
+            u32 sinCosTableIndex = yRotAngle >> (jmaSinShift & 0xFFFF);  
+            plyrToObjVec.z = 
+                plyrToObjVec.x * jmaSinTable[sinCosTableIndex] +
+                plyrToObjVec.z * jmaCosTable[sinCosTableIndex];
+            plyrToObjVec.x = 
+                plyrToObjVec.x * jmaCosTable[sinCosTableIndex] -
+                plyrToObjVec.z * jmaSinTable[sinCosTableIndex];
+        }
+
+        f32 curScale = scale.x;
+        if ((plyrToObjVec.x < -curScale * 100.0f) || (curScale * 100.0f > plyrToObjVec.x)){
+            return false;
+        }
+        curScale = scale.y;
+        if ((plyrToObjVec.y < -curScale * 100.0f) || (curScale * 100.0f > plyrToObjVec.y)){
+            return false;
+        }
+        curScale = scale.z;
+        if ((plyrToObjVec.z < -curScale * 100.0f) || (curScale * 100.0f > plyrToObjVec.z)){
+            return false;
+        }
+    }
+
+    *pos = current.pos;
+    return true;
+}
 
 /* 00000078-00000188       .text _create__Q214daTagAttention5Act_cFv */
 cPhs_State daTagAttention::Act_c::_create() {
-    /* Nonmatching */
+    fopAcM_SetupActor(this, Act_c);
+    mStts.Init(0xFF, 0xFF, this);
+    mSph.Set(sph_check_src);
+    mSph.SetStts(&mStts);
+    return cPhs_COMPLEATE_e;
 }
 
 /* 00000340-00000480       .text _execute__Q214daTagAttention5Act_cFv */
 bool daTagAttention::Act_c::_execute() {
     /* Nonmatching */
+    mSph.SetC(current.pos);
+    mSph.SetR(scale.x * 100.0f);
+    dComIfG_Ccsp()->Set(&mSph);
+    m_b0x290 = true;
+    int iVar1 = daObj::PrmAbstract(this, PRM_1_W, PRM_1_S);
```

Comment:
In the debug map, this `PrmAbstract` call is under `prm_get_Type__Q214daTagAttention5Act_cCFv` which demangles to `daTagAttention::Act_c::prm_get_Type() const`

Those are the empty functions in the header, they will get inlined here so put this `PrmAbstract` call in there and use that `prm_get_Type` function instead.

See [the tutorial on debug maps](https://github.com/zeldaret/tww/blob/main/docs/decompiling.md#reading-rel-debug-maps) and [this example of `PrmAbstract`](https://github.com/zeldaret/tww/blob/4bf91cbfea65258b917cfc5dd60f909b2c9b715d/include/d/actor/d_a_obj_correct.h#L19-L38) for details